### PR TITLE
{chem}[foss/2018a] Molden 5.8

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

File: easybuild/easyconfigs/m/Molden/Molden-5.8-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ Ab Initio packages GAMESS-UK, GAMESS-US and GAUSSIAN and the Semi-Empirical
+ packages Mopac/Ampac"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['ftp://ftp.cmbi.ru.nl/pub/molgraph/molden']
+sources = ['%(namelower)s%(version)s.tar.gz']
+checksums = ['a17bac0901f279e0d89aaf15174875e0695ad7596c5878233946b438d09d93b7']
+
+dependencies = [('X11', '20180131')]
```
Comment (verdurin) [inline easybuild/easyconfigs/m/Molden/Molden-5.8-foss-2018a.eb]: @RvDijk Can you explain the new `X11` dependency? Is it the new docking feature?

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 2 (1 easyconfigs in this PR)
dell.lan - Linux fedora 28, Intel(R) Core(TM) i7-4600U CPU @ 2.10GHz, Python 2.7.15
See https://gist.github.com/43f217fd67312dc26572c095619ac9cd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/80c73565968a7b9ee917b1062a3837e1 for a full test report.

File: easybuild/easyconfigs/m/Molden/Molden-5.8-foss-2018a.eb (hunk 1 above)
Comment (RvDijk) [reply easybuild/easyconfigs/m/Molden/Molden-5.8-foss-2018a.eb]: @verdurin mainly because it was included in #4082.

**Comment by verdurin** [comment]
Going in, thanks @RvDijk!

**Pull request merged**